metecho/api/jobs.py

```diff
@@ -47,7 +47,15 @@ def get_user_facing_url(*, path):
     return furl(f"{scheme}://{domain}").set(path=path).url
 
 
-def project_create_branch(*, project, repository, user, repo_id, originating_user_id):
+def project_create_branch(
+    *,
+    project,
+    repository,
+    repo_id,
+    originating_user_id,
+    user=None,
```

Comment:
@wlonk This is called in the case where `project.branch_name` is not populated... so won't `local_github_checkout(user, repo_id)` fail when `user=None`?